skype/skype-ps/skype/Move-CsUser.md

```diff
@@ -14,7 +14,13 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-Moves one or more user accounts enabled for Skype for Business Server to TeamsOnly (or the reverse). This cmdlet also can be used to move on-premises users from one pool to another.
+Moves one or more user accounts enabled for Skype for Business Server to TeamsOnly (or the reverse). This cmdlet also can be used to move on-premises users from one pool to another. 
+
+**PRE-REQUISITES**
+- Install or upgrade MicrosoftTeams powershell module with version >= 5.2.0
```

Comment:
@ruchiagrawal9221 , This should explicitly state that the below steps are needed only for China environment.

I also don't think we should call out china cloud as Gallatin.
https://learn.microsoft.com/en-us/microsoft-365/admin/services-in-china/services-in-china?view=o365-21vianet
So, need to find the right wordings and add in here.